docs/concepts/artifacts.md

```diff
@@ -178,53 +202,49 @@ if __name__ == "__main__":
     my_fn()
 ```
 
-As you can see, you don't need to create an artifact in a flow run context. You can use it however you like and still get the benefits in the Prefect UI.
+![Table artifact with customer info](/img/ui/table-artifact-info.png)
+As you can see, you don't need to create an artifact in a flow run context.
+You can use it however you like and still get the benefits in the Prefect UI.
```

Comment:
```suggestion
You can use it however you like and still get the benefits in the Prefect UI.
```
in a vanilla Python script, maybe? "however you like" is a touch vague, looking back on this now